.circleci/config.yml

```diff
@@ -11,6 +11,9 @@ jobs:
           keys:
             - v4-test-cargo-cache-linux-{{ arch }}-{{ checksum "Cargo.lock" }}
       - run: sudo apt-get install -y cmake
+      - run:
+          name: Run nginx
+          command:  cargo run -- run examples/nginx/nginx.wasm -- -v
```

Comment:
This will ouptut the nginx version: `nginx version: nginx/1.15.3`
Is this command using the clock contants under the hood? I tested in current `master` and it's running properly. However running `cargo run -- run examples/nginx/nginx.wasm -- -p examples/nginx -c nginx.conf` fails.

I guess we can either remove this integration test for now or make it in a way that really spins up the nginx server.

Reply:
yeah I would like it to actually run the server! Do you have ideas on how to run the server, then exit gracefully? I think a shell script may work, but I'm open to other ideas.

Reply:
I am going to add integration tests in a different PR...

Reply:
Sounds good!